test/e2e/apigatewayv2/tests/test_api.py

```diff
@@ -0,0 +1,292 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Integration tests for the API Gateway V2
+"""
+
+import logging
+import time
+
+import boto3
+import pytest
+import requests
+
+import apigatewayv2.tests.helper as helper
+from apigatewayv2 import SERVICE_NAME, service_marker
+from apigatewayv2.bootstrap_resources import get_bootstrap_resources, AUTHORIZER_FUNCTION_NAME
+from apigatewayv2.replacement_values import REPLACEMENT_VALUES
+from apigatewayv2.tests.helper import ApiGatewayValidator
+from common import k8s
+from common.aws import get_aws_region, get_aws_account_id
+from common.resources import load_resource_file
+
+DELETE_WAIT_AFTER_SECONDS = 10
+UPDATE_WAIT_AFTER_SECONDS = 10
+DEPLOYMENT_WAIT_AFTER_SECONDS = 10
+
+
+@pytest.fixture(scope="module")
+def apigw_validator():
+    return ApiGatewayValidator(boto3.client('apigatewayv2'))
+
+@pytest.fixture(scope="module")
+def test_resource_values():
+    return REPLACEMENT_VALUES.copy()
```

Comment:
These don't need to be a fixture, you can just use them as globally scoped variables within tests?

Reply:
Done.